docs/data-structures.rst

```diff
@@ -97,29 +98,14 @@ Transaction Test (object)
 .. _transaction-results:
 
 Transaction Results (object)
-----------------------------
-
-This is a cousin of the :ref:`gavel-validation-result`.
-
--  general (object) - contains Dredd’s custom messages (e.g. “test was skipped”), formatted the same way like those from Gavel
-
-   -  results (array[:ref:`gavel-error`])
-
--  statusCode (:ref:`gavel-validator-output`)
--  headers (:ref:`gavel-validator-output`)
--  body (:ref:`gavel-validator-output`)
-
-.. _gavel-validation-result:
-
-Gavel Validation Result (object)
 --------------------------------
 
-Can be seen also `here <https://relishapp.com/apiary/gavel/docs/javascript/request-validation#validate>`__ and `here <https://relishapp.com/apiary/gavel/docs/javascript/response-validation#validate>`__.
+Transaction result equals to the validation result of the Gavel validation result. You can read more about the `Gavel validation structure <https://github.com/apiaryio/gavel.js>`__.
```

Comment:
The first sentence _result_ is quite _validation result_ heavy with _validation result_. What about:

```diff
- Transaction result equals to the validation result of the Gavel validation result. You can read more about the `Gavel validation structure <https://github.com/apiaryio/gavel.js>`__.
+ Transaction result equals to the result of the `Gavel <https://github.com/apiaryio/gavel.js>`__ validation library.
```

Reply:
:D That's what happens when you edit a sentence multiple times.